website/content/en/v1.0/upgrading/v1-migration.md

```diff
@@ -626,6 +626,10 @@ Ensure these values are set for all resources **before** upgrading Karpenter.
 Failing to do so will result in Karpenter being unable to operate against those resources.
 For the AWS provider, the group will always be `karpenter.k8s.aws` and the kind will always be `EC2NodeClass`.
 
+### Forceful Expiration
+
+Expiration is now forceful and begins draining as soon as it’s expired without waiting for replacement capacity. Karpenter will start provisioning a replacement as soon as the node is expired and begins draining. This forceful approach means that all expired nodes hat were previously blocked from disruption (due to Pod Disruption Budgets (PDBs), do-not-disrupt pods, or other constraints) will be disrupted simultaneously. The scheduler will then attempt to provision new nodes for the disrupted workloads. This behavior may lead to an increased number of pods in the "Scheduling" state while replacement capacity is being provisioned. 
```

Comment:
> This behavior may lead to an increased number of pods in the "Scheduling" state while replacement capacity is being provisioned

What is a "Scheduling" state? When I read this, I assume it's a [pod's phase](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase), but there is no "Scheduling" phase. If this is referring to a metric, we should refer to it specifically, but I don't think there's a metric that really maps to this.